backend/amd64/selection.ml

```diff
@@ -115,14 +115,13 @@ let pseudoregs_for_operation op arg res =
       ([| rax; rcx |], [| rdx |])
   | Ispecific Irdtsc ->
   (* For rdtsc instruction, the result is in edx (high) and eax (low).
-     Make it simple and force the result in rdx and rax clobbered. *)
-    ([| |], [| rdx |])
+     Make rax and rdx clobbered. *)
+    ([| |], res)
```

Comment:
This match clause is redundant now, maybe?

Reply:
ah, yes, thank you! I've removed it.